docs/manual/manual_ifci_control.rst

```diff
@@ -7,23 +7,86 @@
 IQUART Flight Controller Interface
 **********************************
 
-Vertiq modules can be controlled with the IQUART Flight Controller Interface (IFCI) while running pulsing firmware. A specifc IQUART packet has been created to allow for highspeed control of Vertiq modules. A single packet can include all of the control signals for multiple modules connected to the same UART port on a flight controller. Additionally a telemetry packet can be requested from a specific module every time the packet is sent. An example of a full IFCI packet is shown in the diagram below.
+About IFCI and Control Values
+=================================
+
+Vertiq modules may be controlled through the IQUART Flight Controller Interface (IFCI) when running applicable firmware. Please refer to your module's family page to see if 
+IFCI is supported. The IQUART Flight Controller Interface leverages Vertiq's :ref:`IQUART serial protocol <uart_messaging>` in order to send high speed controls to your modules. 
+These controls are packed into a single message allowing one IFCI packet to send controls to multiple modules connected to the same serial line. 
+The basic structure of an IFCI packet is illustrated below:
 
 .. figure:: ../_static/manual_images/pulsing_propeller/Long_Packed_Control_Message.png
     :align: center
     :alt: Packed Control Message Description
 
     Packed Control Message Description
 
-A general description of IQUART packets can be found in :ref:`uart_messaging`.
+Please note that an IFCI packet does not necessarily need to populate all 16 control values.
+
+Highlighted in orange are the bytes that are necessary in order to send valid IQUART messages, and are not specific to IFCI control.
+
+Highlighted in green are Control Values. A control value is an unsigned 16 bit value that defines a throttle, an X, or a Y command. X and Y commands are only applicable when 
+using Vertiq's :ref:`pulsing firmware <pulsing_module_start_guide>`. Any type of command can occupy any control value, and the location it occupies in an IFCI packet is called its Control 
+Value Index (CVI). For example, suppose you want to control a generic quadcopter. You may place the throttle commands for motor 1 in CVI 0, commands for motor 
+2 in CVI 1, and so on. Where each motor's throttle command is fully configurable, however, and it would be equally valid to place motor 4's throttle command in CVI 0.
+
+In order to bridge the flight controller's commands with the commands accepted by a module, Vertiq modules have their own version of the expected CVIs. For example, 
```

Comment:
I was quite confused by this paragraph originally, I now realize its referring to the fact that you can set the expected CVI values for things on the module side, but somehow the phrasing here really threw me off.

I may do something more like this to replace the first two sentences, just because I found it a bit confusing:

"Because any control value can be stored at any control value index, you must configure what control value indices each of your Vertiq modules should expect to pull its control values from. These must be configured to match where the flight controller expects to store those control values."